content/sensu-go/5.2/installation/install-sensu.md

```diff
@@ -360,6 +360,26 @@ sensuctl config set-namespace default
 
 See the [sensuctl reference][4] for more information about using sensuctl.
 
+### 3. Activate enterprise-only features
+
+Sensu Enterprise offers features designed for enterprises, like LDAP authentication and enterprise-class support.
+To learn more about Sensu Enterprise, [contact the Sensu sales team](http://sensu.io.sales) to schedule a demo.
+
+If you already have a Sensu Enterprise account, log in to your Sensu account at [account.sensu.io](https://account.sensu.io/) and download your license file.
+Then apply your license using sensuctl.
```

Comment:
similar comment to above -- either remove "then" or connect to the preceding sentence